### ISSUE TITLE
Minitest deprecations

### ISSUE THREAD

**Issue by ball-hayden** [Azure/azure-storage-ruby#151]
There are numerous deprecation warnings running the unit test suite under Minitest 5.14.0.

E.g:

```
DEPRECATED: global use of must_equal from /Users/hayden/development/GitHub/azure-storage-ruby/test/unit/core/auth/shared_access_signature_test.rb:119. Use _(obj).must_equal instead. This will fail in Minitest 6.
DEPRECATED: global use of must_be_kind_of from /Users/hayden/development/GitHub/azure-storage-ruby/test/unit/file/file_service_test.rb:1196. Use _(obj).must_be_kind_of instead. This will fail in Minitest 6.
```

**Comment by katmsft** [comment]
This is going to be fixed in the latest release. Please stay tuned.

**Comment by katmsft** [comment]
This issue is addressed in azure-storage-common 2.0.1, please check it out and thanks for reporting.
